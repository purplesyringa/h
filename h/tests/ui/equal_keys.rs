fn main() {
    let meow = h::map!(
        for u8;
        0 => Some(()),
        0 => None,
    );
}
