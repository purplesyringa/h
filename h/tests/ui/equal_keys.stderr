error: this key is equivalent to...
 --> tests/ui/equal_keys.rs:4:9
  |
4 |         0 => Some(()),
  |         ^

error: ...this key
 --> tests/ui/equal_keys.rs:5:9
  |
5 |         0 => None,
  |         ^
